lib/components/mailbox/mailbox.css

```diff
@@ -180,27 +178,100 @@ a:visited {
     color: #fff;
 }
 
-#thead_from {
-    min-width: 100px;
-    max-width: 200px;
+#message_list {
+    table-layout: fixed;
+}
+
+#message_list tbody,
+#message_list tr,
+#message_list td {
+    display: block;
+    box-sizing: border-box;
 }
 
-#thead_subject {
-    min-width: 200px;
+#message_list tbody {
+    width: 100%;
+}
+
+#message_list tr {
+    display: block;
+}
+
+#message_list tr.thead {
+    border-bottom: 2px solid #4b2e4c;
+}
+
+#message_list td {
+    padding: 5px 1.5%;
+    display: inline-block;
+}
+
+.messageItems tr:not(:last-child) {
+    border-bottom: 2px solid #4b2e4c;
+}
+
+.messageItems {
+    border-bottom-left-radius: 5px;
+    height: 241px;
+    overflow-y: auto;
+    overflow-x: hidden;
 }
 
 td.tbody_from,
 td.tbody_subject {
     overflow: hidden;
 }
 
-td #thead_delete,
-td .tbody_delete {
-    width: 60px;
+td.body_subject {
+  word-break: break-word;
 }
 
-ur-button.btn_delete {
-    float: right;
+#thead_from,
+td.tbody_from {
+    width: 20%;
+}
+
+#thead_subject,
+td.tbody_subject {
+    width: 60%;
+}
+
+#thead_delete,
+td.tbody_delete {
+    width: 18%;
+    text-align: right;
+}
+
+#message_list tbody::-webkit-scrollbar {
```

Comment:
It didn't pick up the global styling I saw for scrollbars elsewhere so had to bring it all in here specifically.